### PR TITLE
Blog post about building the lunaconf android app. 

### PR DESCRIPTION
**Pull request by shabab477**
Touches base on:

- The process of coming to decisions regarding the stack
- What we liked using Kotlin
- What we didn't like
- What we would like to have in the future

### CODE REVIEW

**Review by thinkmorestupidless — APPROVED** [review]
I've made some minor suggestions here but on the whole i like the post... it's a bit short and perhaps some more detail would have been nice but if we do more Android work we can follow-up 🤷

**Comment by shabab477** [comment]
> I've made some minor suggestions here but on the whole i like the post... it's a bit short and perhaps some more detail would have been nice but if we do more Android work we can follow-up 🤷

Thank you for the suggestions @thinkmorestupidless . Yeah, its a bit short. The idea behind this post is more about our experience while using Kotlin for android. More posts incoming as we get to architect our app.

File: posts/2022-12-17-leveraging-the-lunaconf-android-app-with-kotlin-and-jetpack-compose.adoc [hunk 1]
```diff
@@ -0,0 +1,75 @@
+= Leveraging the Lunaconf-2022 Android app with Kotlin and Android Jetpack Compose
+shabab477
+v1.0, 2022-12-16
+:title: Leveraging the Lunaconf-2022 Android app with Kotlin and Android Jetpack Compose
```
[the remark this answers is not in the document]
Comment (shabab477) [reply posts/2022-12-17-leveraging-the-lunaconf-android-app-with-kotlin-and-jetpack-compose.adoc]: Right, renamed to `Using Kotlin to build the Lunaconf 2022 App`

File: posts/2022-12-17-leveraging-the-lunaconf-android-app-with-kotlin-and-jetpack-compose.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+ . Use cross-platform frameworks: Work with Flutter, React Native, or Ionic
+ . Use native frameworks: Work with Java or Kotlin
+
+We considered building the app to be cross-platform. Flutter would be great for that. However, Flutter would require us 
+to learn Dartlang. Since the team responsible mainly consists of Java/Scala engineers, something like Kotlin would be perfect for us. 
+Also, developing the app with Kotlin doesn't necessarily mean we are not doing cross-platform. 
+With https://kotlinlang.org/lp/mobile/[Kotlin-Multiplatform], we can easily port our app to support other platforms in later iterations. 
+So we decided going native for the first iteration is the best solution for us.
+
+Now the question is whether we want to build our apps with Kotlin or Java. While Java has had its time, and at Lunatech
```
[the remark this answers is not in the document]
Comment (shabab477) [reply posts/2022-12-17-leveraging-the-lunaconf-android-app-with-kotlin-and-jetpack-compose.adoc]: Right, changed to suggested phrasing. Thanks

**Pull request merged**